### DISCUSSION TITLE
Problem with Azure Data Factory connection with on premise Sql Server

### DISCUSSION THREAD

**Discussion by HamzaLghali** [prisma/prisma#24438] [Q&A]
### Question

Cannot connect to SQL Database. Please contact SQL server team for further support. Server: '', Database: 'dbname', User: 'username'. Check the linked service configuration is correct, and make sure the SQL Database firewall allows the integration runtime to access.
Login failed for user 'username'., SqlErrorNumber=18456,Class=14,State=1,

**Comment by ludralph** [comment]
Hi @HamzaLghali 👋 

Thank you for raising this question.

The error message you encountered, `SqlErrorNumber=18456,Class=14,State=1`, indicates a login failure for the user `username`.  Can you please ensure that the `username` and `password` being used are correct. Double-check for any typos or case sensitivity issues.

You can take a look at this [docs](https://www.prisma.io/docs/orm/overview/databases/sql-server#connection-details) for connecting to SQLServer.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
